src/Stratis.Bitcoin/Stratis.Bitcoin.csproj

```diff
@@ -5,7 +5,7 @@
     <AssemblyTitle>Stratis.Bitcoin</AssemblyTitle>
     <TargetFramework>netstandard2.0</TargetFramework>
     <AssemblyName>Stratis.Bitcoin</AssemblyName>
-    <PackageId>Stratis.Bitcoin</PackageId>
+    <PackageId>Blockcore.FullNode</PackageId>
```

Comment:
I wonder about this, should it not just be Blockcore? or maybe Blockcore.Core but that does't sounds right.

Reply:
Think Blockcore.Core is better, sounds a bit funny but more descriptive. Have updated